simpleviewinc/keg-herkin
# ZEN-611 : Resetting the testName to an empty string so that it doesn'…

…t cache the filename from the previously created file

**Ticket**: https://jira.simpleviewtools.com/browse/ZEN-611

## Context
The newly created test file name was cached when we opened a new window causing user experience issues

## Goal

After a new test file is created, when the user tries to create another one , the test file name should not be cached. Also when the user selects another feature/test file , and then clicks on the 'new file' button , the new window that appears should not have the test file name cached and should be empty

## Updates

* repos/tap/components/modals/createFileModal.js
  * Ensured that the 'testName' variable was reset after the file was created in the modal because it was still holding onto the value set previously

## Testing

Docker image : docker pull ghcr.io/simpleviewinc/keg-herkin:zen-611-filenamecaching

**Setup**

- Pull this PR => keg herkin && keg pr 44
- Navigate to repos/example folder and then start keg herkin => keg herkin start
- Open launched app at http://herkin-zen-611-filenamecaching.local.keghub.io/
- Escape from the pop up window so you are on the main site

**Tests**
* Test 1:
  * Click on the 'Settings' button on the top right hand corner
  * Leaving all fields already selected at default click 'Do it'
  * In the 'Create test File' modal , add a file name and click 'Create test file'
  * Click the button on the left side to open up the sidebar menu
  * Click the new file button on the top right of the sidebar menu
  * When the new 'Create Test File ' modal opens up it should NOT have the earlier file name present or cached

* Test 1:
  * Click on the 'Settings' button on the top right hand corner
  * Leaving all fields already selected at default click 'Do it'
  * In the 'Create test File' modal , add a file name and click 'Create test file'
  * Click the button on the left side to open up the sidebar menu
  * Select any existing feature file 
  * Then click the new 'Create Test File ' modal .It should NOT have the earlier file name present or cached
